src/LinuxTracingIntegrationTests/IntegrationTestPuppet.cpp

```diff
@@ -22,6 +22,10 @@
 // This executable is used by LinuxTracingIntegrationTest to test the generation of specific
 // perf_event_open events. The behavior is controlled by commands sent on standard input.
 
+// Hack: Don't use ORBIT_API_INSTANTIATE as it would redefine `struct orbit_api_v1 g_orbit_api_v1`,
+// which is already defined by the Introspection target.
+void* orbit_api_get_function_table_address_v1() { return &g_orbit_api_v1; }
```

Comment:
Of course let me know if you have a better idea.

Reply:
Maybe namespaces could fix that? But also, totally fine for me to keep it that way.

Reply:
If I put `ORBIT_API_INSTANTIATE` in a namespace then the `orbit_api_get_function_table_address_v1` function gets prefixed by a namespace as well and it doesn't get considered in the whole `liborbit.so`-injection process, which is correct, because it would otherwise return the address of a different `g_orbit_api_v1`, the one that also ended up in a namespace.

Reply:
Right, I was actually thinking of putting the introspection into a namespace. But that probably also does not work.

Reply:
That's probably the same issue but seen from the opposite side: `orbit_api_v1 g_orbit_api_v1` defined by introspection has to be in the global namespace to match the declaration in `Orbit.h`.